issue-416/README.md

```diff
@@ -21,6 +21,41 @@ Stating @CamilleLegeron
 >
 > Camille
 
+## Glossary
+
+<details>
+  <summary>Acronyms</summary>
+  <dl>
+    <dt>MH</dt><dd>Must Have</dd>
+    <dt>SH</dt><dd>Should Have<dd>
+    <dt>NTH</dt><dd>Nice To Have</dd>
+  </dl>
+</details>
+
+<details>
+  <summary>Vocabulary</summary>
+  <dl>
+    <dt>Synchronized table</dt><dd>The table which fetches its data from an external table, and gets its data and columns synchronized with this external table</dd>
+    <dt>External table</dt><dd>Table of another document of the synchronized table</dd>
+  </dl>
+</details>
+
+## Acceptance Criteria
+
+- AC1 - MH: A user can create a table whose table is synchronized with an external document
+- AC2 - MH: The synchronization is made when the user requests it.
+  - AC2.1 - MH: The user sees what data have been changed through a diff
+  - AC2.2 - SH: The user also sees what columns are removed or added
```

Comment:
Deleted a column of an external document synchronized could break formula, so I think it could be a MH ?

Reply:
Actually, I even think to lower the priority of the diff, as it seems the codebase of the diff seems to be fragile as reported by Paul:
https://github.com/gristlabs/grist-core/issues/416#issuecomment-1652486834

I agree with Paul to iterate and start with something even more modest, and maybe present the feature as in beta